styx-common/src/main/java/com/spotify/styx/util/ShardedCounter.java

```diff
@@ -181,7 +181,7 @@ public ShardedCounter(Storage storage, CounterSnapshotFactory counterSnapshotFac
   /**
    * Returns a recent snapshot, possibly read from inMemSnapshot.
    */
-  private CounterSnapshot getCounterSnapshot(String counterId) {
+  public CounterSnapshot getCounterSnapshot(String counterId) {
```

Comment:
Why was this made public? It seems to be used below for `getLimit` but that is already available as a public method on `ShardedCounter`.

Reply:
My bad, not public yet. Let's fix that one to be public, instead of publicizing CounterSnapshot, which is a relatively deeper implementation detail.

Reply:
I remember it wasn’t that straightforward to just make getLimit public. Maybe I’m wrong. It’s been a while.